tests/phpunit/integration/Core/Authentication/Setup_Test.php

```diff
@@ -167,8 +167,9 @@ public function test_handle_action_setup_start__wp_error( $has_credentials ) {
 		$user_id = $this->factory()->user->create( array( 'role' => 'administrator' ) );
 		wp_set_current_user( $user_id );
 
-		$context = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE, new MutableInput() );
-		$setup   = new Setup( $context, new User_Options( $context ), new Authentication( $context ) );
+		$context                      = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE, new MutableInput() );
+		$setup                        = new Setup( $context, new User_Options( $context ), new Authentication( $context ) );
+		$oauth_proxy_failed_help_link = $setup->get_oauth_proxy_failed_help_link();
```

Comment:
As mentioned above, I don't think this test should be referencing `get_oauth_proxy_failed_help_link()`...
```suggestion
```